src/app/containers/main/index.jsx

```diff
@@ -65,12 +67,20 @@ class Main extends Component {
     }
 
     searchRequest(searchTerm, offset) {
+        this.setState(initialState);
+
         if (searchTerm) {
+            const error = {
+                message: `Searching for "${searchTerm}"...`,
+                imageUrl: loadingImage,
+            };
+            this.setState({error});
```

Comment:
Yea, let's add a `state.status` and maybe even have it be `state.status.isError`